### PR TITLE
Add new parameters in gpt xblock - EDLY-5564

### PR DESCRIPTION
**Pull request by taimoor-ahmed-1**
**Description:** This PR adds new parameters for chatgpt xblock.

**JIRA:** https://edlyio.atlassian.net/browse/EDLY-5564

### CODE REVIEW

File: chatgptxblock/chatgptxblock.py [hunk 1]
```diff
@@ -48,6 +48,25 @@ class ChatgptXBlock(StudioEditableXBlockMixin, XBlock):
...
+        help="The maximum number of tokens to generate.",
+    )
+
+    start_text = String(
+        display_name="Start Text",
+        default="",
+        scope=Scope.settings,
+        help="Text to append after the user's input."
+    )
+    temperature =Float(
```
Comment (marslanabdulrauf) [inline chatgptxblock/chatgptxblock.py]: missing space after =

File: chatgptxblock/chatgptxblock.py [hunk 2]
```diff
@@ -48,6 +48,25 @@ class ChatgptXBlock(StudioEditableXBlockMixin, XBlock):
...
+        default=0.5,
+        scope=Scope.settings,
+        help="Controls Radomness."
```
Comment (marslanabdulrauf) [inline chatgptxblock/chatgptxblock.py]: typo, Randomness*

**Review by marslanabdulrauf — APPROVED** [review]
Please review minor changes

File: chatgptxblock/chatgptxblock.py (hunk 2 above)
Comment (taimoor-ahmed-1) [reply chatgptxblock/chatgptxblock.py]: done

File: chatgptxblock/chatgptxblock.py (hunk 1 above)
Comment (taimoor-ahmed-1) [reply chatgptxblock/chatgptxblock.py]: done

**Pull request merged**